src/QMCDrivers/DMC/WalkerControlMPI.cpp

```diff
@@ -196,7 +211,7 @@ int WalkerControlMPI::branch(int iter, MCPopulation& pop, FullPrecRealType trigg
 void WalkerControlMPI::determineNewWalkerPopulation(int Cur_pop,
                                                     int NumContexts,
                                                     int MyContext,
-                                                    const std::vector<int>& NumPerNode,
+                                                    std::vector<int>& num_per_node,
```

Comment:
This routine is shared between production and unified drivers. I'm cautious on this API change. Please update documentation of this function.